Makefile

```diff
@@ -0,0 +1,12 @@
+build:
+	npm install
+	npm run build
+
+test:
+	make build
+	mkdir testing && cd testing && npm init -y && npm install ..
+
+clean:
+	rm -R testing
+
+
```

Comment:
A small detail but please remove this line break (only one line break at the end of the file)